src/components/block/index.ts

```diff
@@ -814,6 +814,9 @@ export default class Block extends EventsDispatcher<BlockEvents> {
         contentNode = $.make('div', Block.CSS.content),
         pluginsContent = this.toolInstance.render();
 
+    wrapper.classList.add(Block.CSS.wrapper + "-"+this.id);
+    wrapper.dataset.blockId = this.id;
```

Comment:
the more preferred way is to add `data-id` attribute instead.

Also, please add a change log and a test case in `block-ids.cy.ts`